src/answers-search-bar.js

```diff
@@ -215,11 +215,13 @@ class AnswersSearchBar {
     this._handlePonyfillCssVariables(parsedConfig.disableCssVariablesPonyfill)
       .finally(() => {
         this._onReady();
-        const impressionEvent = createImpressionEvent({
-          verticalKey: parsedConfig.search?.verticalKey,
-          standAlone: true
-        });
-        this._analyticsReporterService.report(impressionEvent, { includeQueryId: false });
+        if (this._analyticsReporterService) {
+          const impressionEvent = createImpressionEvent({
+            verticalKey: parsedConfig.search?.verticalKey,
+            standAlone: true
+          });
+          this._analyticsReporterService.report(impressionEvent, { includeQueryId: false });
```

Comment:
nit: I believe we can do something like `this._analyticsReporterService?.report()`, if we want to avoid writing the entire if statement for this, but it's really not necessary

Reply:
oh yeah I just didn't want to create the event if we didn't need to